From 87a630136db3860ea60e31875afca444316a7212 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 11 Nov 2020 10:35:44 +0100
Subject: [PATCH 1/2] config.mak.uname: remove unused the NO_R_TO_GCC_LINKER
 flag
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The NO_R_TO_GCC_LINKER flag was still being on some platforms. It
hasn't been used since my 0f50c8e32c8 ("Makefile: remove the
NO_R_TO_GCC_LINKER flag", 2019-05-17).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 config.mak.uname | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/config.mak.uname b/config.mak.uname
index c7eba69e54e..8d7b96f8b9b 100644
--- a/config.mak.uname
+++ b/config.mak.uname
@@ -636,7 +636,6 @@ ifneq (,$(wildcard ../THIS_IS_MSYSGIT))
 	prefix =
 	INSTALL = /bin/install
 	EXTLIBS += /mingw/lib/libz.a
-	NO_R_TO_GCC_LINKER = YesPlease
 	INTERNAL_QSORT = YesPlease
 	HAVE_LIBCHARSET_H = YesPlease
 	NO_GETTEXT = YesPlease
@@ -669,7 +668,6 @@ else
 			-fstack-protector-strong
 		EXTLIBS += -lntdll
 		INSTALL = /bin/install
-		NO_R_TO_GCC_LINKER = YesPlease
 		INTERNAL_QSORT = YesPlease
 		HAVE_LIBCHARSET_H = YesPlease
 		NO_GETTEXT =
@@ -695,7 +693,6 @@ ifeq ($(uname_S),QNX)
 	NO_MKDTEMP = YesPlease
 	NO_NSEC = YesPlease
 	NO_PTHREADS = YesPlease
-	NO_R_TO_GCC_LINKER = YesPlease
 	NO_STRCASESTR = YesPlease
 	NO_STRLCPY = YesPlease
 endif
-- 
2.29.2.222.g5d2a92d10f8

